src/ui-shell/header/header-menu.component.ts

```diff
@@ -23,7 +23,8 @@ import { HeaderItemInterface } from "./header-navigation-items.interface";
 				[href]="href"
 				tabindex="0"
 				aria-haspopup="true"
-				[attr.aria-expanded]="expanded">
+				[attr.aria-expanded]="expanded"
+				(click)="navigate()">
```

Comment:
Why are we using an anchor in header menu? I noticed that all of the other carbon libs are using `<a>` as well 🤔